src/test/ui/fn_once-moved.stderr

```diff
@@ -1,12 +1,15 @@
 error[E0382]: use of moved value: `debug_dump_dict`
   --> $DIR/fn_once-moved.rs:21:5
    |
+16 |         for (key, value) in dict {
+   |                             ---- dict moved here
```

Comment:
This is great! My only concern is that, when you read this message, the `dict moved here` seems a bit out of the blue. I am torn between trying to give more context in this label vs trying to move the label down below.

Also, as a minor style nit, we should put `dict` in backticks:

```
`dict` moved here
```

Basically, it seems highly likely to me that people will get confused between the *closure* being moved twice and *variable `dict` being moved twice*.